Goder.BL/Goder.BL.csproj

```diff
@@ -8,11 +8,13 @@
     <Folder Include="DTO\" />
     <Folder Include="Hubs\" />
     <Folder Include="MappingProfiles\" />
-    <Folder Include="Services\" />
+	<Folder Include="Services\" />
   </ItemGroup>
 
   <ItemGroup>
     <PackageReference Include="AutoMapper" Version="10.1.1" />
+    <PackageReference Include="Microsoft.AspNetCore" Version="2.2.0" />
```

Comment:
You don't need `"Microsoft.AspNetCore` and `Microsoft.AspNetCore.Mvc` in BL project